### DISCUSSION TITLE
Latest tag not available in maven repository

### DISCUSSION THREAD

**Discussion by fullopt** [google/closure-compiler#4184] [Q&A]
Hi,

why is the latest tag (v20240609) not available in [maven repository](https://mvnrepository.com/artifact/com.google.javascript/closure-compiler)?

Thank you.

**Comment by gkdn** [comment]
There was a problem with CI that broke Maven uploads so we needed to disable daily releases. We are also considering to change the cadence of releases but it is not finalized yet. We will post an update when it is finalized. Sorry for any inconveniences.

**Reply by fullopt** [reply]
Would it be possible to upload the latest version to maven?

**Reply by kozi** [reply]
Is there anything new on this?
